### DISCUSSION TITLE
Can't use finger gestures on trackpad to zoom

### DISCUSSION THREAD

**Discussion by nvarghese-rm** [SafeExamBrowser/seb-mac#169] [Q&A]
Just wanting to check if there is any reason why the the finger gestures for zooming on a trackpad is not working with SEB Mac. 

You are able to scroll and click using the trackpad but not be able pinch to zoom. It works in Safari as expected when I check. 

Is that expected for SEB Mac or do I need to make some other configuration change?

**Comment by danschlet** [comment]
Probably it works in the modern WebView and not in the classic one. I'll have a look at that tomorrow.

**Reply by nvarghese-rm** [reply]
I have tested in SEB MAC v3.0 and v2.1.4 and observe the issue.

**Reply by danschlet** [reply]
Pinch-to-zoom actually had to be activated in code, which I did in the [pre-release SEB 3.2pre5](https://github.com/SafeExamBrowser/seb-mac/releases). But it definitely only works when SEB uses the modern WebView, which is only the case when specific SEB settings are used, see the [release notes of SEB 3.0](https://github.com/SafeExamBrowser/seb-mac/releases/tag/3.0).
